.coderabbit.yaml

```
@@ -34,7 +34,19 @@ reviews:
   auto_reply: true
 
-tone_instructions: "스프링 모범 사례, SOLID 원칙, 성능 최적화, 보안 관련 사항, 적절한 JPA 사용법, 자바 코딩 표준을 강조한다"
+tone_instructions: "
+  ## 기본 규칙
+  - 코드의 잠재적인 버그나 성능 문제를 찾아주세요.
+  - 스프링 모범 사례, SOLID 원칙에 맞게 개선할 점이 있다면 제안해 주세요.
+  - 성능 최적화가 가능한 로직이 존재한다면 알려주세요.
+  - 클린 코드 원칙에 따라 개선할 점이 있다면 제안해 주세요.
+  - 변경 사항을 요약해서 설명해 주세요.
+  - 코드가 업데이트 되었을 때 클로드 코드로 작성된 이전 리뷰와 다른 내용을 작성해 주세요.
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)